common/src/main/java/org/mcaccess/minecraftaccess/mixin/ChatComponentMixin.java

```diff
@@ -0,0 +1,27 @@
+package org.mcaccess.minecraftaccess.mixin;
+
+import org.mcaccess.minecraftaccess.Config;
+import org.mcaccess.minecraftaccess.utils.WorldUtils;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.GuiMessageTag;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.components.ChatComponent;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MessageSignature;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.sounds.SoundSource;
+
+@Mixin(ChatComponent.class)
+public class ChatComponentMixin {
```

Comment:
As per #596, all mixins that are not accessors should be package private abstract classes.
```suggestion
abstract class ChatComponentMixin {
```